src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,115 @@
+
+.cards {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  @media screen and (width:1200px) {
+    .cards {
+      margin: 50px 129px 73px;
+    }
+  }
+  .card {
+    width: 200px;
+    background: #fff;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 0;
+    margin-right: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-top: 50px;
+    }
+
+    &_img {
+      display: block;
+      height: 134px;
+      width: 160px;
+      margin: 0 auto 40px;
+    }
+
+    &_text {
+      font-style: normal;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      color: #060b35;
+      margin-bottom: 4px;
+    }
+
+    &_product_code {
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+      margin-bottom: 16px;
+    }
+
+    &_stars_reviews {
+      display: flex;
+      margin-bottom: 24px;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    &_reviews {
+      color: #060b35;
+      font-size: 10px;
+      line-height: 14px;
+    }
+
+    &_price {
+      display: flex;
+      margin-bottom: 16px;
+      justify-content: space-between;
+    }
+
+    &_item {
+      width: 31px;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &_number {
+      width: 50px;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+
+    &_buy {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-sizing: border-box;
+      margin-bottom: 16px;
+    }
+
+    &_buy--decoration {
+      border-radius: 5px;
+      background: $color;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      color: #fff;
+      text-decoration: none;
+      padding: 12px;
+    }
+
+    &_buy:hover {
```

Comment:
Add a smooth transition to the hover effect by adding 'transition: all 0.3s ease-in-out;' to the '&_buy--decoration' selector.